PolyEngine/Core/Src/String.cpp

```diff
@@ -0,0 +1,207 @@
+#include"CorePCH.hpp"
+
+#include"String.hpp"
+
+using namespace Poly;
+
+size_t Poly::StrLen(const char* str) {
+	size_t len = 0;
+	while (str[len] != 0)
+		++len;
+	return len;
+}
+
+
+String::String(const char* data) {
+	size_t length = StrLen(data);
+	Data.Resize(length + 1);
+	std::memcpy(Data.GetData(), data, sizeof(char) * length);
+	Data[length] = 0;
+}
+
+String::String(const String& rhs) {
+	*this = rhs;
+}
+
+String::String(String&& rhs) {
+	*this = String(std::move(rhs));
+}
+
+
+String* String::From(int var) {
+	return new String(std::to_string(var).c_str());
```

Comment:
Please remove every single on new operator from this implementation. None of them is necessary.